media/js/firefox/features/landing.js

```diff
@@ -6,4 +6,10 @@
     'use strict';
 
     Mozilla.LazyLoad.init();
+
+    // data-link-position is dynamic due to L10n trickle-in
+    // may eventually be needed for other feature types on this page
```

Comment:
This is a nice solution 👍 

My only question is whether it may be worth doing this now for all three lists? It may not be obvious why two lists have the attribute but the first does not. This is maybe going a bit beyond the scope of this bug, but it might not be that much more code to write?